knative/knative-serving-install/base/deployment.yaml

```diff
@@ -221,7 +221,7 @@ spec:
         sidecar.istio.io/inject: "false"
       labels:
         app: networking-istio
-        serving.knative.dev/release: "v0.11.1"
+        serving.knative.dev/release: "v0.11.2"
```

Comment:
I don't see knative serving version bump in this PR? is it in the scope?

Reply:
It is a minor version bump of 0.11 release and only networking istio deployment image is changed with the probing fix.